src/env.h

```diff
@@ -1005,6 +1009,17 @@ struct SnapshotData {
   // read only space. We use native_module::CodeCacheInfo because
   // v8::ScriptCompiler::CachedData is not copyable.
   std::vector<native_module::CodeCacheInfo> code_cache;
+
+  static std::unique_ptr<SnapshotData> New();
```

Comment:
Why not a regular constructor? The constructor currently doesn't communicate any sort of failure.

Reply:
Because we can't use list initializers when there's a non-default constructor.